test/functional-tests/features/search.feature

```diff
@@ -0,0 +1,51 @@
+@search @functional
+Feature: Set of scenarios to test functionality of search filters on the search page
+
+  Background:
+    Given I have logged in
+
+  Scenario: selecting the jurisdiction search drop down changes the main banner title to that jurisdiction name
+    Given I am on the search page
+    And the banner title matches that of the currently selected jurisdiction
+    When I change the jurisdiction search drop down option
+    Then the banner title matches that of the currently selected jurisdiction
+
+  Scenario: dynamic search filters are displayed for all data types
+    Given a case type containing every field type exists
+    And I am on the search page
+    When I select the 'Case type' drop down option for dynamic filters
+    Then a dynamic filter of every datatype will be displayed
+
+  Scenario: reset button clears drop down options and removes all dynamic filters
+    Given a case type containing every field type exists
+    And I am on the search page and selected case type
```

Comment:
we should try to avoid a step that is doing 2 things in the same line. we could do the actual selecting of the case type in the step before or after perhaps

Reply:
Yes I can do that. Was hoping to reduce number of steps.